store/postgres/src/store.rs

```diff
@@ -6,16 +6,19 @@ use diesel::sql_types::Text;
 use diesel::{delete, insert_into, select, update};
 use filter::store_filter;
 use futures::sync::mpsc::{channel, Sender};
+use lru_time_cache::LruCache;
```

Comment:
I was thinking last night: I wonder what cache Leo will pick for this, LRU would make sense. And here we go 😍

Reply:
LRU does seem good.

Mental notes I made here: A built-in cache is of course the fastest possible, but the cache size will not really scale with the number of query nodes. Since queries have no affinity to particular query nodes (and we probably want to avoid that), then it's likely that all query nodes will have very similar contents in their caches. In fact, having more query nodes means it will take more initial queries to start getting consistent cache hits across all nodes.

Having it still better than not having it, but we should be aware of the limitations.

Reply:
True. I think we can live with that.